SingularityS3Base/src/main/java/com/hubspot/singularity/s3/base/config/SingularityS3Configuration.java

```diff
@@ -131,7 +131,7 @@ public void setLocalDownloadPath(String localDownloadPath) {
     this.localDownloadPath = localDownloadPath;
   }
 
-  public long getS3ChunkDownloadTimeoutMillis() {
+  public Long getS3ChunkDownloadTimeoutMillis() {
```

Comment:
in order to more cleanly call `intValue()` (aws sdk takes an int). Was debating if it was cleaner to keep the `TimeUnit.SECONDS.toMillis(30)`so the default is readable and have to do `long` -> `int`, or to move it all to int. Reading it now the second probably makes more sense, I'll update